### DISCUSSION TITLE
width and height setting seems not working now on CovideoX1.5 model?

### DISCUSSION THREAD

**Discussion by DStarEpoch** [THUDM/CogVideo#523] [Q&A] [answered]
I generate video by running 
python inference/cli_demo.py  with CovideoX1.5 t2v model 
try to set --wdith and --height to various number, however output video shape is always 1360x768.
It seems shape setting is not working now?

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
Supported, I am not sure if you have passed the parameters correctly, you need to ensure that your resolution is less than 1360x768
